module-2/backend/week_four.md

```diff
@@ -10,14 +10,57 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+
+  A cookie is a token that holds user information that can be recognized by the requesting server. 
```

Comment:
This is a good description, but careful with the word `token`. It has a very particular meaning that you'll get more into in M3.